src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -308,6 +308,7 @@ BtreeLBAManager::_alloc_extent(
   pladdr_t addr,
   paddr_t actual_addr,
   laddr_t intermediate_base,
+  extent_len_t intermediate_length,
```

Comment:
nit: drop the unused parameters

Reply:
Specifially, intermediate_base/length looks unused.

Reply:
Done